src/field/field2d.cxx

```diff
@@ -89,7 +89,7 @@ Field2D::Field2D(const Field2D& f) : Field(f.fieldmesh), // The mesh containing
   *this = f; //This line is probably not required as we init data from f.data above.
 }
 
-Field2D::Field2D(BoutReal val) : Field(nullptr), deriv(nullptr) {
+Field2D::Field2D(BoutReal val, Mesh * msh) : Field(msh), deriv(nullptr) {
   boundaryIsSet = false;
 
   fieldmesh = mesh;
```

Comment:
This reverts `fieldmesh` back to the global `mesh` -- should probably be updated to mirror the other `Field2D` ctors

Reply:
Ah well spotted!

Reply:
Thanks, fixed in 227c93a